source/resource/qx/tool/website/partials/head.html

```diff
@@ -15,6 +15,7 @@
     <title>{{= it.title || it.site.title }}</title>
     <meta name="description" content="{{= it.excerpt || it.site.excerpt || "" }}">
     <link rel="stylesheet" type="text/css" href="/qooxdoo.css">
+    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/bluebird/3.5.5/bluebird.js"></script>
```

Comment:
It would be better to store the file locally and load it from there so that it works in offline situations.

Reply:
done